app/components/util/HowDoYouFeel.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from 'react'
+import { createUseStyles } from 'react-jss'
+import cx from 'classnames'
+
+const responseOptions = ['Awesome!', 'Just Okay', 'Unsatisfied']
+
+export default function HowDoYouFeel(props) {
+    //Isolate props and set initial state
+    const { disabled, className, onDone, ...otherProps } = props
+    const styleClasses = rankingStyleClasses(props)
+    const classes = buttonStyles()
+    const combinedClassName = cx(classes.Howdoyoufeel, className)
+    const textClasses = textStyles()
+    const textClassName = cx(textClasses.textStyle)
+    const onSelection = e => {
+        onDone({ valid: true, value: e.target.value })
+    }
+    return (
+        <>
```

Comment:
Make this `<div className={className} {...otherProps} >` then do not use props.className oe otherProps anywhere else. 
This means combinedClassName goes away.